aspnetcore/fundamentals/configuration/index.md

```diff
@@ -190,6 +190,8 @@ Call <xref:Microsoft.Extensions.Hosting.HostBuilder.ConfigureAppConfiguration*>
 `AddCommandLine` has already been called by `CreateDefaultBuilder`. If you need to provide app configuration and still be able to override that configuration with command-line arguments, call the app's additional providers in <xref:Microsoft.Extensions.Hosting.HostBuilder.ConfigureAppConfiguration*> and call `AddCommandLine` last.
 
 ```csharp
+// using Microsoft.Extensions.Configuration;
```

Comment:
Why is this commented out?  When it has to be because it's out of place in the source code in order to be included in a snippet, there's no choice but to comment out, but I'm not sure it makes sense when it's unnecessary because it's inline code in the Markdown file.